layers/+emacs/emacs-info/README.org

```diff
@@ -0,0 +1,10 @@
+#+TITLE: Emasc Info Imrprovements
+
+#+TAGS: emacs|layer
+
+* Description
+  This layer improves navigation for Emacs Info mode
+** Features:
```

Comment:
Please add a blank line between headlines and the last line of the previous paragraph.
Also add a list of provided features please.

I would suggest:
```suggestion
** Features:
- Intuitive =transient state=
```